test/Interop/Cxx/ergonomics/implicit-computed-properties-module-interface.swift

```diff
@@ -265,3 +264,19 @@
 // CHECK-NEXT:    func getValue() -> Int32
 // CHECK-NEXT:    mutating func setValue(_ i: Int32)
 // CHECK-NEXT: }
+
+// CHECK:      struct SnakeCaseGetter {
+// CHECK-NEXT:    init()
+// CHECK-NEXT:    init(foo: Int32)
+// CHECK-NEXT:    var foo: Int32 { mutating get set }
+// CHECK-NEXT:    func get_foo() -> Int32
+// CHECK-NEXT:    mutating func set_foo(_ i: Int32)
+// CHECK-NEXT: }
+
+// CHECK:      struct SnakeCaseUTF8Str {
+// CHECK-NEXT:    init()
+// CHECK-NEXT:    init(utf8String: Int32)
+// CHECK-NEXT:    var utf8String = Int32
```

Comment:
This should be different. So, I don't think the tests are running on your local machine. 

When you run `lit` are you getting any errors or anything? Are you sure this test runs?

Reply:
Hey zoe, i think youre right. I may have some issues with my local environment. I was getting several failures with tests, more recently i ran
 `utils/run-test --build-dir ../build/Xcode-DebugAssert/swift-macosx-x86_64 ./test/Interop/Cxx/ergonomics`

and had this result: 
`error: no such module 'StdlibUnittest'
import StdlibUnittest
       ^
Failed Tests (2):
  Swift(macosx-x86_64) :: Interop/Cxx/ergonomics/implicit-computed-properties-module-interface.swift
  Swift(macosx-x86_64) :: Interop/Cxx/ergonomics/implicit-computed-properties.swift
`

Reply:
I assume you're running with ninja? Make sure to build `ninja swiftStdlibUnittest-macosx`.

Reply:
That should be done by the build script, though. Try passing `--test` to the buildscript. Once it starts running tests you can cancel the build and proceed with `lit`.

Reply:
> That should be done by the build script, though. Try passing `--test` to the buildscript. Once it starts running tests you can cancel the build and proceed with `lit`.

Hey @zoecarver, everything should be updated, tests are passing now